modules/core/misc/python/pyopencv_umat.hpp

```diff
@@ -0,0 +1,38 @@
+#ifdef HAVE_OPENCV_CORE
+
+#include "opencv2/core/mat.hpp"
+
+typedef std::vector<Range> vector_Range;
+
+CV_PY_TO_CLASS(UMat);
+CV_PY_FROM_CLASS(UMat);
+CV_PY_TO_ENUM(UMatUsageFlags);
+
+static bool cv_mappable_to(const Ptr<Mat>& src, Ptr<UMat>& dst)
+{
+    //dst.reset(new UMat(src->getUMat(ACCESS_RW)));
+    dst.reset(new UMat());
+    dst->allocator = &g_numpyAllocator;
```

Comment:
Why is allocator replacement necessary for `UMat`? It is not intended for processing via numpy .

Reply:
@alalek Now fixed